src/foam/u2/ExportModal.js

```diff
@@ -99,6 +105,13 @@ foam.CLASS({
           .start(this.DATA_TYPE).end()
           .start().addClass('label').add('Response').end()
           .start(this.NOTE).addClass('input-box').addClass('note').end()
+          .add(
+            self.slot(function(dataType) {
+              if(dataType == 'CSV') {
```

Comment:
Follow FOAM style guide for spacing:
 if ( dataType == 'CSV' ) {

Reply:
fixed